backend/taggle-web/src/main/java/kr/taggle/security/config/WebSecurityConfig.java

```diff
@@ -27,7 +35,8 @@ protected void configure(final HttpSecurity http) throws Exception {
 
         http
                 .authorizeRequests()
-                        .antMatchers("/", "/signin", "/fonts/**", "/img/**","/css/**", "/js/**", "/taggle-favicon.ico", "/h2-console/**", "/api/**").permitAll()
+                        .antMatchers("/", "/signin", "/login/**", "/h2-console/**", "/css/**", "/img/**", "/js/**", "/api/v1/users/me").permitAll()
```

Comment:
```suggestion
                        .antMatchers("/", "/signin", "/h2-console/**", "/css/**", "/img/**", "/js/**", "/api/v1/users/me").permitAll()
```
자체적인 로그인 페이지를 사용하니 /login 접근 권한은 없어도 무방할 것 같아요